examples/compound-assignment-operators/compound_assignment_operators.bal

```diff
@@ -0,0 +1,46 @@
+import ballerina/io;
+
+public function main() {
+    int value = 1;
+    io:println("initial value: ", value);
+
+    // Compound `add` operator.
+    value += 7;
+    io:println("add: ", value);
```

Comment:
Shall we improve the output?

Maybe the statements themselves
```suggestion
    io:println("value += 7: ", value);
```
or including the values also
```suggestion
    io:println("add 7: ", value);
```